templates/config/controller/deployment.yaml.tpl

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller
+  name: ack-system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ack-{{ .ServiceAlias}}-controller
+  namespace: ack-system
+  labels:
+    control-plane: controller
```

Comment:
is it possible to one more label to help query service specific controller ? something like 
`controller: ack-{{ .ServiceAlias}}-controller`

Reply:
Sure, that's a great idea. :)